Description: Don't output pip warnings, otherwise some doctests fail
Author: Ximin Luo <infinity0@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/sage
+++ b/sage/sage
@@ -16,6 +16,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
+export PIP_DISABLE_PIP_VERSION_CHECK=1
 
 # Set SAGE_ROOT to the location of the sage install, i.e. the directory
 # containing this shell script.  If unset, we will try to figure it out
